log/log.go

```diff
@@ -24,15 +24,15 @@ func Init(verbose bool) {
 }
 
 func Error(fmt string, args ...string) {
-	log.Printf("[ERROR] "+fmt, args)
+	log.Printf("ERROR \n"+fmt, args)
```

Comment:
Are you sure about the newline position?

Reply:
yup, fixed.